Dockerfile

```diff
@@ -128,4 +128,6 @@ ENV MEILI_LOG_LEVEL="WARN"
 EXPOSE 7878
 EXPOSE 80
 
+HEALTHCHECK --interval=5m --timeout=10s --start-period=10s --retries=3 CMD curl --fail http://localhost/ || exit 1
```

Comment:
what this do?

Reply:
if 5 minutes, it hits localhost to see if it is up. If it isn't then the container will fail.